stdlib/Statistics/src/Statistics.jl

```diff
@@ -73,7 +73,29 @@ function mean(f, itr)
     end
     return total/count
 end
-mean(f, A::AbstractArray) = sum(f, A) / length(A)
+
+"""
+    mean(f::Function, A::AbstractArray, [dims])
+
+Apply the function `f` to each element of `A` across `dims` and take the mean.
+
+```jldoctest
+julia> mean(√, [1, 2, 3])
+1.3820881233139908
+
+julia> mean([√1, √2, √3])
+1.3820881233139908
+
+julia> mean(√, [1 2 3; 4 5 6], dims = 2)
```

Comment:
No spaces around `=` for keyword arguments (here and a bunch of other places, I won't comment on each)
```suggestion
julia> mean(√, [1 2 3; 4 5 6], dims=2)
```